### PR TITLE
[sparkle] - chore: UI polish

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at taking the `sparkle` changes from this https://github.com/dust-tt/dust/pull/8429 to split it between a front and sparkle PR.

## Risk

Low

## Deploy Plan

Publish `sparkle`

### CODE REVIEW

File: sparkle/src/components/Tabs.tsx [hunk 1]
```diff
@@ -56,7 +58,8 @@ const TabsTrigger = React.forwardRef<
       <TabsPrimitive.Trigger
         ref={ref}
         className={cn(
-          "s-border-0 s-border-b-2 s-border-primary-800/0 s-pb-1 disabled:s-pointer-events-none data-[state=active]:s-border-primary-800",
+          "s-h-[43px]",
```
Comment (flvndvd) [inline sparkle/src/components/Tabs.tsx:61]: Can we avoid arbitrary value? Where is this one coming from?

File: sparkle/src/components/EmptyCTA.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import { Button, ButtonProps } from "@sparkle/components/";
+import { cn } from "@sparkle/lib/utils";
+
+interface EmptyCTAProps extends React.HTMLAttributes<HTMLDivElement> {
+  action: React.ReactNode;
+  message: string;
+}
+
+const EmptyCTA = React.forwardRef<HTMLDivElement, EmptyCTAProps>(
```
Comment (flvndvd) [inline sparkle/src/components/EmptyCTA.tsx:11]: OOC, what will this used for?

**Review by flvndvd — APPROVED** [review]

File: sparkle/src/components/EmptyCTA.tsx (hunk 2 above)
Comment (JulesBelveze) [reply sparkle/src/components/EmptyCTA.tsx:11]: @Duncid can you shed some light here?

File: sparkle/src/components/Tabs.tsx (hunk 1 above)
Comment (JulesBelveze) [reply sparkle/src/components/Tabs.tsx]: don't like it either ✅ 🤫

File: sparkle/src/components/EmptyCTA.tsx (hunk 2 above)
Comment (Duncid) [reply sparkle/src/components/EmptyCTA.tsx:11]: Yes :)
It's a new component to standardize all of these:
image

**Pull request merged**